scripts/api.sh

```diff
@@ -97,7 +97,8 @@ api_withdraw() {
 # $1 = node api endpoint
 api_get_balances() {
   local origin=${1}
```

Comment:
Might use double-quotes to be consistent and prevent splitting.

Reply:
Agreed. I wanted to clean up those files separately, plenty of things not nice in there.